<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>341</width>
     <height>501</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QRadioButton" name="passwordRadioButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Use Passphrase</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="showPwCheckBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Show Password</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QLineEdit" name="passwordLineEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Password</string>
          </property>
          <property name="statusTip">
           <string>Password</string>
          </property>
          <property name="whatsThis">
           <string>Password</string>
          </property>
          <property name="text">
           <string>Password</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::PasswordEchoOnEdit</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="retypeLineEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Retype Password</string>
          </property>
          <property name="statusTip">
           <string>Retype Password</string>
          </property>
          <property name="whatsThis">
           <string>Retype Password</string>
          </property>
          <property name="text">
           <string>Retype Password</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="keyfileRadioButton">
        <property name="text">
         <string>Use Key File</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item>
           <widget class="QLineEdit" name="keyFileLineEdit">
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="keyFileBrowseToolButton">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="keyFileNewToolButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../asset/img/new.png</normaloff>../../asset/img/new.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="md5CheckBox">
          <property name="text">
           <string>Generate md5 checksum</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>LUKS Volume File</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="newVolLineEdit">
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="newVolBrowseToolButton">
            <property name="toolTip">
             <string>Browse</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="newVolNewToolButton">
            <property name="toolTip">
             <string>New File</string>
            </property>
            <property name="text">
             <string>+</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../asset/img/new.png</normaloff>../../asset/img/new.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Size of LUKS Volume</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QSpinBox" name="sizeNumberSpinBox">
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="sizeUnitComboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>GiB</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>MiB</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>KiB</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Format:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="formatComboBox">
          <item>
           <property name="text">
            <string>ext4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ntfs</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>fat16</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>fat32</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Security</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::HLine</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Volume Settings</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="canclePushButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>510</y>
     <width>92</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancle</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="okPushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>92</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>newVolLineEdit</tabstop>
  <tabstop>newVolBrowseToolButton</tabstop>
  <tabstop>newVolNewToolButton</tabstop>
  <tabstop>sizeUnitComboBox</tabstop>
  <tabstop>formatComboBox</tabstop>
  <tabstop>passwordLineEdit</tabstop>
  <tabstop>retypeLineEdit</tabstop>
  <tabstop>keyfileRadioButton</tabstop>
  <tabstop>keyFileLineEdit</tabstop>
  <tabstop>keyFileBrowseToolButton</tabstop>
  <tabstop>keyFileNewToolButton</tabstop>
  <tabstop>md5CheckBox</tabstop>
  <tabstop>okPushButton</tabstop>
  <tabstop>canclePushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
